.travis.yml

```diff
@@ -141,9 +152,11 @@ script:
         if npm run danger -- ci --dangerfile ./dangerfile-android.js | grep -q "hasAndroidFile" ; then
           cd android
           ./gradlew clean assembleRelease --quiet -PbuildRuntimeApi=true ${GRADLE_ABI} -PapachePackageName="false" || exit 1
-          ./gradlew assembleRelease --quiet ${GRADLE_ABI} -PbuildRuntimeApi=false -Pjsc_url="https://raw.githubusercontent.com/apache/incubator-weex/release/0.26/android/sdk/weex_sdk-0.26.0.aar" -Paar_name="weex_sdk" -PJSInterpolatorName="JavaScriptCore" -PapachePackageName="false" || exit 1
           ./gradlew :weex_sdk:assembleRelease --quiet -PbuildRuntimeApi=true ${GRADLE_ABI} -PapachePackageName="true" || exit 1
-          ./gradlew :weex_sdk:assembleRelease --quiet  ${GRADLE_ABI} -PbuildRuntimeApi=false -Pjsc_url="https://raw.githubusercontent.com/apache/incubator-weex/release/0.26/android/sdk/weex_sdk-0.26.0.aar" -Paar_name="weex_sdk" -PJSInterpolatorName="JavaScriptCore" -PapachePackageName="true" || exit 1
+          if [[ "$ABI" != "x86_64" ]]; then
+            ./gradlew assembleRelease --quiet ${GRADLE_ABI} -PbuildRuntimeApi=false -Pjsc_url="https://raw.githubusercontent.com/apache/incubator-weex/release/0.26/android/sdk/weex_sdk-0.26.0.aar" -Paar_name="weex_sdk" -PJSInterpolatorName="JavaScriptCore" -PapachePackageName="false" || exit 1
```

Comment:
Is Pjsc_url still useful? 
Didn't understand why the weex_sdk v0.26 aar is used.

Reply:
As we delete `JavaScriptCore.so` from our code, and some developers still want it for some reasons.

So, with `Pjsc_url=0.26xxx`, weex will use `JavaScriptCore.so` as JS interpretator. This is a edge case in Travis CI, and never happened in a release code.

Developers have the freedom of choosing what JS interpretator they want, and I'd like our test case can cover some of their choice on JS interpretator.

Reply:
`JavaScriptCore.so` is still working now, but no body is maintaining it. As time goes and things happen(like it doesn't support x86_64), it will be abondoned eventually.

Reply:
thanks